dashboard/public/openapi.yaml

```diff
@@ -3170,7 +3307,50 @@ paths:
             type: string
             pattern: .+
       requestBody:
-        $ref: "#/components/requestBodies/v1alpha1UpdatePackageRepositoryRequest"
```

Comment:
Don't know why this is being removed, but after the change in #4708, it isn't generating the associated requestBody separately :S

Reply:
Hmm, maybe the length of the URL being shorter? No idea.